tests/test_propagate.py

```diff
@@ -88,25 +88,3 @@ def test_exact_propagation():
     # Exact is a bit worse here since it requires a lot of padding.
     # TODO: Find better test case.
     assert rel_error < 1e-2
-
-
-def test_propagate():
```

Comment:
Why is this gone?

Reply:
Because we got rid of the `propagate` method since it was not jitable. People need to call one of the three propagate methods directly with the padding calculator completely removed from the loop.